### DISCUSSION TITLE
How to implement gradient background in v2.0 CssInJs

### DISCUSSION THREAD

**Discussion by devtrix** [skeletonlabs/skeleton#1991] [Q&A] [answered]
Previously in v1 I had this in my theme called simple.
`
 [data-theme='simple'] {
    background-image:
        radial-gradient(at 0% 0%, rgba(var(--color-tertiary-900) / 0.33) 0px, transparent 50%),
        radial-gradient(at 98% 1%, rgba(var(--color-tertiary-800) / 0.33) 0px, transparent 50%);

}
`
How do I implement this in the current theme file.  I created the simple-theme.ts file and ported over the variables.  
I am not sure where or how to add the above.
Any help is appreciated.

**Comment by endigo9740** [comment, marked as answer]
Drop the styles as you have them in `/src/app.postcss`. They'll work as expected as long as that theme is active.

**Reply by devtrix** [reply]
You are the man. Thank you for continuing to be patient. I had that suspicion but thank you for clarifying. Have a great weekend.
